tests/tracking/test_tracking.py

```diff
@@ -243,3 +219,23 @@ def test_with_startrun():
     assert run_info.status == RunStatus.from_string("FINISHED")
     assert t0 <= run_info.end_time and run_info.end_time <= t1
     assert mlflow.active_run() is None
+
+
+def test_parent_create_run(temporary_tracking_uri):
+    parent_run = mlflow.start_run()
+    with pytest.raises(Exception, match='To start a nested run'):
```

Comment:
Nit: an existing test (`test_no_nested_run`) already verifies this - maybe we can just delete `test_no_nested_run`

Reply:
also nice, will remove `test_no_nested_run`.